src/SchedulerSubsystem/SchedulerSystem.java

```diff
@@ -181,18 +184,18 @@ public static void main(String[] args) {
      	elevatorSystem.start();		
      	
      	//  scheduler <-> floor (thread 1)
-		sendPort = 24;
-		receivePort = 23;
+		sendPort = FloorSystem.FLOOR_RECEIVE_PORT;
+		receivePort = FloorSystem.FLOOR_SEND_PORT;
 		ByteBufferCommunicator floorBufferCommunicator2 = new ByteBufferCommunicator(sendPort, receivePort);
 		
 		// scheduler <-> elevator (thread 2)
-		sendPort = 70;
-		receivePort = 69;
+		sendPort = ElevatorSystem.RECEIVE_PORT;
+		receivePort = ElevatorSystem.SEND_PORT;
 		ByteBufferCommunicator elevatorBufferCommunicator2 = new ByteBufferCommunicator(sendPort, receivePort);
 		
-		
-		// TODO Remove hardcode of 3
-		SchedulerSystem schedulerSystem = new SchedulerSystem(elevatorBufferCommunicator2, floorBufferCommunicator2, 3);
+
+		// TODO Might need to give faultBufferCommunicator
+		SchedulerSystem schedulerSystem = new SchedulerSystem(elevatorBufferCommunicator2, floorBufferCommunicator2, ElevatorSystem.NUM_ELEVATORS);
```

Comment:
Can we reference the elevatorSystem from scheduler? If the idea is that they run on different systems idk if we can do this.